crates/utils/src/settings/mod.rs

```diff
@@ -41,8 +39,8 @@ impl Settings {
   }
 
   /// Returns the config as a struct.
-  pub fn get() -> Self {
-    SETTINGS.read().expect("read config").to_owned()
+  pub fn get() -> &'static Self {
+    &SETTINGS
   }
```

Comment:
You can probably remove this get() method, and make SETTINGS public instead. Also, i dont think Settings::init() needs to be public.